main/migrations/sqls/20210208213802-AddDomainEventTable-up.sql

```diff
@@ -0,0 +1,11 @@
+/* Replace with your SQL commands */
+CREATE TABLE domain_event

```

Comment:
We are using `jsonb` and `timestamptz` that aren't supported by the db-migrate data types. Hence we are using sql here.
I also prefer to create partitions from the get go to avoid painful data migration later when this event table gets too big.
What do you think about it? Should we defer or create partitions (range partitions based on created_at)?

Reply:
Logically the concept looks same to me except at different levels (nosql being built with pure scaling in mind).